person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services?
+    @age >= 18 and @parent_permissiond == true
+  end
```

Comment:
- [ ] can_use_services must return true if a person is of age **or** if they have permission from parents. Currently, you have used `and`, instead you have to use `or`.

Reply:
Ok. I will try to change it.